src/tree/JobTreeItem.ts

```diff
@@ -0,0 +1,86 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Octokit } from '@octokit/rest';
+import { ActionsGetJobForWorkflowRunResponseData } from '@octokit/types';
+import * as moment from 'moment';
+import * as prettyMs from 'pretty-ms';
+import { AzExtTreeItem, AzureParentTreeItem, IActionContext, TreeItemIconPath } from "vscode-azureextensionui";
+import { createOctokitClient } from '../commands/github/createOctokitClient';
+import { Conclusion, Status } from '../gitHubTypings';
+import { convertConclusionToVerb, convertStatusToVerb, getRepoFullname } from '../utils/gitHubUtils';
+import { treeUtils } from "../utils/treeUtils";
+import { ActionTreeItem } from './ActionTreeItem';
+import { IAzureResourceTreeItem } from './IAzureResourceTreeItem';
+import { StepTreeItem } from './StepTreeItem';
+
+export class JobTreeItem extends AzureParentTreeItem implements IAzureResourceTreeItem {
+
+    public static contextValue: string = 'azureStaticJob';
+    public readonly contextValue: string = JobTreeItem.contextValue;
+    public parent: ActionTreeItem;
+    public data: ActionsGetJobForWorkflowRunResponseData;
+
+    constructor(parent: ActionTreeItem, data: ActionsGetJobForWorkflowRunResponseData) {
+        super(parent);
+        this.data = data;
+    }
+
+    public get iconPath(): TreeItemIconPath {
+        return treeUtils.getActionIconPath(<Status>this.data.status, <Conclusion>this.data.conclusion);
+    }
+
+    public get id(): string {
+        return this.data.id.toString();
+    }
+
+    public get name(): string {
+        return this.data.name;
+    }
+
+    public get label(): string {
+        return this.name;
+    }
+
+    public get description(): string {
+        if (this.data.conclusion !== null) {
+            const elapsedMs: number = this.completedDate.getTime() - this.startedDate.getTime();
+            return `${convertConclusionToVerb(<Conclusion>this.data.conclusion)} ${moment(this.completedDate).fromNow()} in ${prettyMs(elapsedMs)}`;
```

Comment:
So I wanted to confirm with GitHub, but apparently it doesn't support any language other than English: https://github.blog/2010-07-13-github-in-your-language/

I feel like we might want to localize this string because I'm not sure this format is standard in say, Chinese or something.